web_external/js/models/DatasetModel.js

```diff
@@ -6,6 +6,10 @@ minerva.models.DatasetModel = minerva.models.MinervaModel.extend({
         files: null
     },
 
+    isWmsSource: function () {
```

Comment:
I don't like this, but I totally understand why you did it, and it makes sense given that we don't have clear separation in the UI b/w sources and datasets.

I'll add an issue to refactor this once the source list appears in the UI.

https://github.com/Kitware/minerva/issues/113